osu.Game/Beatmaps/Drawables/DifficultyIcon.cs

```diff
@@ -170,4 +201,48 @@ public bool SetContent(object content)
             protected override void PopOut() => this.FadeOut(200, Easing.OutQuint);
         }
     }
+
+    public class DifficultyIconWithCounter : DifficultyIcon
```

Comment:
xmldoc, this is all too confusing to exist without it.

also should be in own file?

Reply:
why not call this `GroupedDifficultyIcon` or something?